### ISSUE TITLE
404 from nginx

### ISSUE THREAD

**Issue by kennyparsons** [alextran1502/immich#40]
I have an instance up and was able to create a user with the curl commands. I can also ping the server with 
```bash
curl http://localhost:2283/server-info/ping
{"res":"pong"}%
```

However, when i open my browser to the app (`http://localhost:2283/`), I get:
```
{"statusCode":404,"message":"Cannot GET /","error":"Not Found"}
```
I'm running this on Docker Desktop for Mac (Intel).

**Comment by mikegrogan** [comment]
In talking with Alex recently, the website isn't ready yet. Just the app. I tested on Android running the APK and was able to load it up and login, as well as backup photos w/o too many issues. I got the 404 trying to run the website as well.

**Comment by kennyparsons** [comment]
Well then I should open another issue. I couldn't get flutter to compile the iOS app.

**Comment by alextran1502** [comment]
@kennyparsons This is currently the backup tool for your photos and videos from your mobile phone. So there is no web client. 

It might evolve to a gallery app but it is not my initial goal of this application. 

I am planning to have TestFlight and the app on the Apple Store once the mobile app is ready to be released, my apologies for the inconvenience if you are using an iPhone.
